ports/servo/build.rs

```diff
@@ -83,3 +89,10 @@ fn android_main() {
     println!("cargo:rustc-link-lib=log");
     println!("cargo:rustc-link-lib=android");
 }
+
+fn windows_main() {
+    let mut res = winres::WindowsResource::new();
+    res.set_resource_file("windows.rc");
```

Comment:
When working on the code last night, I noticed there is a conflicting file setting the icon, it's [support/windows/Servo.wxs.mako](https://github.com/servo/servo/blob/master/support/windows/Servo.wxs.mako#L74)